libcontainer/container_nabla.go

```diff
@@ -266,14 +272,57 @@ func (c *nablaContainer) exec() error {
 func (c *nablaContainer) destroy() error {
 	c.state.InitProcessPid = 0
 	c.state.Status = Stopped
-	if err := os.RemoveAll(c.root); err != nil {
+
+	execInput := &ll.ExecCreateInput{}
```

Comment:
I think it is more Go standard to have something like:
```
	execInput := &ll.ExecCreateInput{
		ExecGenericInput: ll.ExecGenericInput{
			ContainerRoot: c.root,
			Config:        c.config,
			ContainerId:   c.id,
			FsState:       &c.state.FsState,
			ExecState:     &c.state.ExecState,
		},
	}
```

But I'm not entirely sure on it.

Reply:
Good catch! Resolved in 80f8a080abe330f90abf62e5060c9c87e2a7ffa3